### PR TITLE
Xgb streamlined

### PR DESCRIPTION
**Pull request by benwulfe**
More robust gdrive mounting handles exceptions and creates a local folder as a substitute.

### CODE REVIEW

File: raster.py
Comment (VimWizardVersace) [inline raster.py:1]: Maybe just except ModuleNotFoundError instead of catch all? This exception type is guaranteed to be thrown if it's local.

**Review by VimWizardVersace — CHANGES_REQUESTED** [review]

File: raster.py
Comment (benwulfe) [reply raster.py:1]: fixed

**Review by rothn — APPROVED** [review]

File: raster.py [hunk 1]
```diff
@@ -70,13 +69,19 @@ def get_animations_path(filename: str) -> str:
...
+  if not os.path.exists(GDRIVE_BASE):
     # Access data stored on Google Drive
     if GDRIVE_BASE:
+      try:
         from google.colab import drive
         drive.mount(GDRIVE_BASE)
+      except ModuleNotFoundError:
+        print('WARNING, GDRIVE NOT MOUNTED! USING LOCAL FS!!!')
+
+      if not os.path.exists(GDRIVE_BASE):
```
Comment (erickzul) [inline raster.py:81]: Unless I'm missing something we should test this case.

**Review by erickzul — CHANGES_REQUESTED** [review]

File: raster.py (hunk 1 above)
Comment (benwulfe) [reply raster.py:81]: it is tested in raster_test by triggering an error during the run thru

**Pull request merged**